src/Symfony/Component/HttpKernel/DependencyInjection/FragmentRendererPass.php

```diff
@@ -58,15 +58,19 @@ public function process(ContainerBuilder $container)
                 throw new \InvalidArgumentException(sprintf('Service "%s" must implement interface "%s".', $id, $interface));
             }
 
+            if (!$tags) {
+                $tags = array(array());
+            }
```

Comment:
how is this possible ? If the service has no tags, it will not be returned by findTaggedServiceIds